packages/ui-kit/src/components/TimeSeries/utils.ts

```diff
@@ -221,11 +221,11 @@ interface GetScalesOptions {
   styles?: ChartStyles
   granularity: TimeSeriesGranularity | null
   isFormatted: boolean
-  isStatic: boolean
+  zone: string
 }
 
 export function getScales(options: GetScalesOptions) {
-  const { styles, granularity, isFormatted, isStatic } = options
+  const { styles, granularity, isFormatted } = options
```

Comment:
Maybe it is a little bit confusing if we destructure some fields and others we don't
```suggestion
  const { styles, granularity, isFormatted, zone } = options
```

Reply:
IMO, we should stop destructuring like this. It is not so bad to write `options.x`, `options.y` in places.